jsonschema/cli.py

```diff
@@ -251,7 +260,24 @@ def load(_):
                 raise _CannotLoadFile()
         instances = ["<stdin>"]
 
-    validator = arguments["validator"](schema)
+    if arguments["base_uri"] is None:
+        resolver = None
+    elif "http:" in arguments["base_uri"] or "https:" in arguments["base_uri"]\
+            or "urn:" in arguments["base_uri"]:
+        resolver = RefResolver(
+            base_uri=arguments["base_uri"],
+            referrer=schema,
+        )
+    else:
+        file_prefix = "file:///{}/" if "nt" == os.name else "file://{}/"
+        resolver = RefResolver(
+            base_uri=file_prefix.format(
+                os.path.abspath(arguments["base_uri"])
```

Comment:
Is it this that's causing file to be split out separately? We shouldn't call `abspath` ourselves unless we have good reason, but we should think it through to find out. If the person using the CLI wants an absolute path, why can't they just pass one?